### DISCUSSION TITLE
Removing highlight formatting from text

### DISCUSSION THREAD

**Discussion by cpg-jmowens** [outline/outline#7434] [General]
It seems like all the other formatting options for text (bold, italics, strike, code block, headings) have a click on/click off function in the pop up bar. But the highlight formatting option, once on, has no obvious way to remove it (other than Ctrl+Z your work if it was the last thing you did, or delete the entire word and type it again without formatting). This feels like a big oversight so I wonder if I'm missing something obvious. Has anyone encountered this issue as well and can advise?

**Comment by tommoor** [comment]
You can select the same highlight color again to remove it, we should probably add an option for "No highlight" in that menu though

**Reply by cpg-jmowens** [reply]
Aha. I thought I'd tried that, thank you. I think a no highlight option would be more obvious/clear for users but this answer solves my problem now. Appreciate the quick reply.

**Reply by tommoor** [reply]
<img width="318" alt="image" src="https://github.com/user-attachments/assets/dda69736-4f88-4a5d-acaf-bfd11c3ea645">

The option has been added and will be in the next release, also available on cloud hosted Outline immediately :)
